Frameworks/UIKit/XamlControls.h

```diff
@@ -121,6 +121,12 @@ WXCGrid* CreateLabel();
 WXCTextBlock* GetLabelTextBlock(WXCGrid* labelGrid);
 
 ////////////////////////////////////////////////////////////////////////////////////
+// ScrollViewer
+////////////////////////////////////////////////////////////////////////////////////
+WXCGrid* CreateScrollViewer();
+WXCScrollViewer* GetScrollViewer(WXCGrid* labelGrid);
```

Comment:
It seems strange that we have a GetScrollView method that doesn't return a ScrollView instead it returns a grid.  Should this be named something like GetScrollViewRootElement?